src/core/components/accordion/index.tsx

```diff
@@ -48,16 +52,25 @@ interface AccordionRowProps extends Props {
 	label: string;
 	hideToggleLabel?: boolean;
 	children: ReactNode;
+	onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
 }
 
 const NoJsRow = ({
 	label,
 	hideToggleLabel = false,
 	children,
+	onClick,
```

Comment:
I don't think we need this prop on the NoJsRow component – it doesn't look like it's being used.

Reply:
I picked it out as otherwise there was a Typescript error from spreading out the remaining props onto the div (as the onClick type signature is expecting a click event from a button element). If there's a better way to handle this in TS then happy to make any changes!

Reply:
Good point! Let's roll with this for now 👍